exercises/reverse-string/README.md

```diff
@@ -0,0 +1,106 @@
+# Reverse String
+
+The introductory exercise to arrays and strings. Just reverse input string.
```

Comment:
I would rephrase this to lose the "arrays" bit, as that is a language-specific implementation detail. Maybe just focus on the actual exercise:

> Given an input string, return its reversed string.